client.go

```diff
@@ -44,10 +44,11 @@ func (c *Client) SubmitTx(ctx context.Context, tx []byte) /* TxResponse */ error
 	return errors.New("method SubmitTx not implemented")
 }
 
-func (c *Client) SubmitPFD(ctx context.Context, namespaceID [8]byte, data []byte, gasLimit uint64) (*TxResponse, error) {
+func (c *Client) SubmitPFD(ctx context.Context, namespaceID [8]byte, data []byte, fee int64, gasLimit uint64) (*TxResponse, error) {
 	req := SubmitPFDRequest{
 		NamespaceID: hex.EncodeToString(namespaceID[:]),
 		Data:        hex.EncodeToString(data),
+		Fee:         fee,
```

Comment:
Since `GasPrice` is generally used along with `GasLimit`, is migration to `Fee` on purpose?

Reply:
go-cnc is 1:1 mapping for gateway API exposed by celestia node. This changes mimics their code. See: https://github.com/celestiaorg/celestia-node/commit/4734aac019e5d786be1148f8cbd0bc84ebc392a9